doc: Update docs with Cellular monitor and Serial Terminal apps (#12395)
nrfconnect/sdk-nrf: samples/cellular/at_client/README.rst

The AT Client sample acts as a proxy for sending directives to the nRF9160 modem using AT commands.
This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.

> You can initiate the commands from a terminal or the `nRF Connect Serial Terminal`_, which is an application implemented as part of `nRF Connect for Desktop`_.

Reviewer: ```suggestion
You can initiate the commands from a terminal such as the `nRF Connect Serial Terminal`_, which is an application implemented as part of `nRF Connect for Desktop`_.
```
Serial Terminal is a terminal emulator, so using "or" here is a bit off.

> You can initiate the commands from a terminal such as the `nRF Connect Serial Terminal`_, which is an application implemented as part of `nRF Connect for Desktop`_.

Reviewer 2: ```suggestion
You can initiate the commands manually from a terminal such as the `nRF Connect Serial Terminal`_, or visually using `Cellular Monitor`_. Both applications are part of `nRF Connect for Desktop`_.
```
